Automattic/jetpack
# Forms: Style customization options for the MC/SC Buttons style

Fixes #30525

## Proposed changes:

* Add style customization options for the MC/SC Buttons style

![image](https://user-images.githubusercontent.com/7811225/236941005-d2d3c4ff-bb37-4ec4-bf44-b19a7a6fc9c8.png)

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Make sure the Forms package is enabled

* Create a post and add a Form block
* Add a Multiple Choice and/or a Single Choice field and change the style to Button
* Play with the new Sidebar style settings and check if they work as expected
* Make sure to check the published page too
